.github/scripts/plugins.py

```diff
@@ -183,13 +185,22 @@ def collect_data_for_plugin(plugin: Plugin, file_groups: FileGroups) -> bool:
     """
     repo = plugin.get("repo")
     branch = plugin.get("branch", "master")
-    manifest = get_plugin_manifest(repo, branch)
+    current_name = str(plugin.get("name"))
 
-    return collect_data_for_plugin_and_manifest(plugin, manifest, file_groups)
+    try:
+        manifest = get_plugin_manifest(repo, branch)
+        return collect_data_for_plugin_and_manifest(plugin, manifest, file_groups)
+
+    except Exception as err:
+        print(f'ERROR processing plugin {current_name}. Error message: {err}')
+        plugin_is_valid = False
+        add_file_group(file_groups, "error", f"{current_name}")
+        return plugin_is_valid
 
 
 def collect_data_for_plugin_and_manifest(plugin: Plugin, manifest: PluginManifest, file_groups: FileGroups) -> bool:
     # the cast to str is to silence: error: Item "None" of "Optional[Any]" has no attribute "split"
+
```

Comment:
The new blank line here separates the comment from the cast that it is referring to... 
It's only a small thing, but is likely to cause confusion down the line.
Best to remove the blank line, I feel.